book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] `add_rental method should receive `person` and `date` as parameter. Also the book itself should be added as parameter.
`
def add_rental(person, date)
...
end
`

Reply:
Done